src/screens/application/index.js

```diff
@@ -62,15 +63,17 @@ const Application = () => {
             </Modal>
 
             {isThemeReady && isRenderRequire && (
-              <ApplicationScreen
-                isConnected={isConnected}
-                locale={locale}
-                toastNotification={toastNotification}
-                isReady={isReady}
-                isDarkTheme={isDarkTheme}
-                rcOffer={rcOffer}
-                foregroundNotificationData={foregroundNotificationData}
-              />
+              <Host>
```

Comment:
@aliseyalvi may be we should place <Host /> in `src/index.js` That seems to be the appropriate place for app wide initialisation as intl and persisteGate is also initialised that.